### PR TITLE
feat: thorchain LP RUNE wallet capability detection for sym withdraws

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Does what it says on the box - sym withdraws are initiated with a RUNE Tx, so we need to add feature detection not to let wallets that do not support RUNE (e.g MM) continue with sym withdraws. 

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A, spotted while testing THORChain LP.

Low, ensure sym withdraws are still available on native

> What protocols, transaction types or contract interactions might be affected by this PR?

## Testing

- Sym withdraws are blocked on MM
- Sym withdraws are happy on native

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

- This diff

- Develop

image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
* **#6369** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6369?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gomesalexandre and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx [hunk 1]
```diff
@@ -766,6 +777,42 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
 
   const divider = useMemo(() => <StackDivider borderColor='border.base' />, [])
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (woodenfurniture) [inline src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx:780]:
```suggestion
```

Comment (gomesalexandre) [reply src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx:780]: bad vim

**Review by woodenfurniture — APPROVED** [review]
LGTM, native sym withdraw working 
https://viewblock.io/thorchain/tx/16E49AA92733D2C1CFBC3548E47ED454F4E884AEC065C880C2C476F386A757C7

and MM sym withdraw not possible 🍻

**Pull request merged**